show-location-history/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.esri.arcgismaps.sample.showlocationhistory">
```

Comment:
```suggestion
```
Remove package name from Manifest and added to Gradle file according to Android [build changes](https://developer.android.com/studio/preview/features#namespace-dsl)